subprojects/docs/src/snippets/files/misc/groovy/build.gradle

```diff
@@ -1,16 +1,24 @@
+import java.nio.file.Files
+
 // tag::mkdir-example[]
 tasks.register('ensureDirectory') {
+    // Store target directory into a variable to avoid project reference in the configuration cache
```

Comment:
This is also exposed in the guide, I've updated the wording to avoid mentioning non-configuration-cache-supported `Project.mkdirs`.